src/styles/admin-core/content-blocks/_enclosed-content.scss

```diff
@@ -41,52 +38,54 @@ $component: "c-block-enclosed-content";
 		&__card {
 			a {
 				text-decoration: none;
+				width: 100%;
 			}
 
+			background-color: $white;
+			border: 1px solid #adadad66;
```

Comment:
don't use colors directly
we should always use color variables or create a variable if it doesn't exist